amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/dashboard/model/ConsumerInfo.java

```diff
@@ -48,4 +48,12 @@ public long getNextSnapshotId() {
   public void setNextSnapshotId(long nextSnapshotId) {
```

Comment:
We don’t seem to need this method. In my opinion, the member variables of this class can be marked as final, WDYT?